examples/active_learning/run.py

```diff
@@ -12,7 +12,11 @@
 import random
 from math import nan
 from pathlib import Path
-from typing import override
+
+try:
+    from typing import override  # Python 3.12+
+except ImportError:
+    from typing_extensions import override  # noqa: UP035
```

Comment:
As sad as it sounds, if we commit to python 3.10, we should not carry code for future versions around. We do the migration as soon as we can drop 3.10.

Maybe we can discuss whether there is no option of upgrading python to a newer version in person...

```suggestion
from typing_extensions import override
```